schemas/oic.wk.p-schema.json

```diff
@@ -95,6 +95,16 @@
           "maxLength": 64,
           "readOnly": true,
           "description": "Manufacturer's defined information for the platform. The content is freeform, with population rules up to the manufacturer"
+        },
+       "nct": {
+          "type": "array",		  
+          "items": {
+            "type": "integer",
+			"description": "The network connectivity type based on IANAIfType value as defined by: https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib"
+          },
+          "minItems": 1,
+		  "readOnly": true,
```

Comment:
Should there be a range?  Are negative integers allowed?

Reply:
The property value follows IANAifType assigned numbers and IANAifType are positive integers from 1 to 294, it seems the numbers keep growing (https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib)